File: 4344 평균은 넘겠지/4344.py
testCount = int(input())
for _ in range(0, testCount):
    inputs = list(map(int, input().split(' ')))
    avg = 0
    for i in range(0, inputs[0]):
        avg += inputs[i + 1]

    avg /= inputs[0]

    count = 0
    for i in range(0, inputs[0]):
        if avg < inputs[i + 1]:
            count += 1
    print("%.3f%%" % round(count / inputs[0] * 100, 3))

File: 4344 평균은 넘겠지/4344.js
let fs = require('fs');
let inputs = fs.readFileSync('/dev/stdin').toString().split('\n');

let n = parseInt(inputs[0]);
for (let i = 1; i <= n; ++i) {
    let inputsSplits = inputs[i].split(' ');
    let n = parseInt(inputsSplits[0]);
    let avg = 0;
    for (let j = 1; j <= n; ++j) {
        avg += parseInt(inputsSplits[j]);
    }

    avg /= n;

    let count = 0
    for (let j = 1; j <= n; ++j) {
        if (avg < inputsSplits[j]) {
            ++count;
        }
    }

    console.log((count / n * 100).toFixed(3) + "%");
}